ios-deploy.xcodeproj/project.pbxproj

```diff
@@ -237,7 +255,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "echo \"\\\"# AUTO-GENERATED - DO NOT MODIFY\\\"\" > src/ios-deploy/lldb.py.h\nawk '{ print \"\\\"\"$0\"\\\\n\\\"\"}' src/scripts/lldb.py >> src/ios-deploy/lldb.py.h\n";
+			shellScript = "echo \"\\\"# AUTO-GENERATED - DO NOT MODIFY\\\\\\n\\\"\" > src/ios-deploy/lldb.py.h\nawk '{ print \"\\\"\"$0\"\\\\n\\\"\"}' src/scripts/lldb.py >> src/ios-deploy/lldb.py.h\n";
```

Comment:
Why add a newline here? It shouldn't cause any issues, but I'm wondering if this does anything.

Reply:
Well it's been more than a year, I really don't remember :D

Reply:
Overall this PR seems fine... I'm mainly worried about having to update other build systems to include that extra shell script when it seems like we could not add it.